.github/workflows/ci.yml

```diff
@@ -295,7 +295,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: nightly-2022-08-12
+          toolchain: nightly-2022-11-05
```

Comment:
This likely requires rerunning the compile tests with `TRYBUILD=overwrite` + committing all the changes.